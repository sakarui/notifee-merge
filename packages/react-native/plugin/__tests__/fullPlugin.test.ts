import withNotifee from '../src';
import { ExpoConfig } from '@expo/config-types';

const dummyConfig: ExpoConfig = {
  name: 'dummy',
  slug: 'dummy',
  _internal: {
    projectRoot: 'dummy-path',
  },
};

describe('Expo plugin', () => {
  test('should run the plugin without error', () => {
    const resultConfig = withNotifee(dummyConfig);
    expect(resultConfig).toMatchSnapshot();
  });
});
