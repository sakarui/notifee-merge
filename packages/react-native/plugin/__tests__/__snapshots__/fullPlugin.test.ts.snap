// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expo plugin should run the plugin without error 1`] = `
Object {
  "_internal": Object {
    "projectRoot": "dummy-path",
  },
  "mods": Object {
    "android": Object {
      "projectBuildGradle": [Function],
    },
  },
  "name": "dummy",
  "slug": "dummy",
}
`;
